src/app/admin/apply/new/_components/questionTypes/index.tsx

```
@@ -0,0 +1,174 @@
+                  onClick={() => handleRemoveOption(i)}
+                />
+              )}
+            </Flex>
+          </Flex>
+        ))}
+      </Radio>
+      {canAddOption && !readOnly && (
+        <button
+          onClick={handleAddOption}
+          className="mt-1 cursor-pointer text-gray-300"
+        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)